homeassistant/components/ezviz/config_flow.py

```diff
@@ -56,7 +58,7 @@ async def _validate_and_create_auth(self, data):
         try:
             await self.hass.async_add_executor_job(client.login)
 
-        except (PyEzvizError, HTTPError) as err:
+        except (PyEzvizError, HTTPError, ConnectionError) as err:
```

Comment:
The library should raise library specific errors only.

Reply:
Separated Library specific errors from general connection and invalid host/url/ip errors.

Reply:
Busy fixing. I see what you mean.

Reply:
Exceptions are now raised by library specific errors.